### ISSUE TITLE
Typo on line 111 - Neural Networks

### ISSUE THREAD

**Issue by jacksusank** [rtealwitter/datamining2026spring#29]
I think this should be a partial derivative instead of a regular derivative:

Old Line: "By the multivariate chain rule, the derivative of the loss function with respect to $u$ is given by"

Proposed Fix: "By the multivariate chain rule, the partial derivative of the loss function with respect to $u$ is given by"

https://github.com/rtealwitter/datamining2026spring/blob/4ab71393f59e05fd1ad92b1a427756007888f03a/notes/NeuralNetworks.qmd#L111

**Comment by rtealwitter** [comment]
+0.25 sure
